proc_nutation.py

```diff
@@ -33,7 +33,6 @@
         fl.image(s)
         s.ift('t2') # make sure everything is in the same domain
         #}}}
-    fl.show();quit()
```

Comment:
this guy was a really big problem -- it allowed for problematic code that came after it.